src/routes/index.svelte

```diff
@@ -73,23 +50,82 @@
     let loading = false
     let highlightedSymbolIndex = null
 
+    let isDocumentReady = false
+
     onMount(() => {
+        let DEFAULT_OPTIONS = []
```

Comment:
Change variable name to `selectedOptions` or something as it is no longer default options and no longer a constant